airflow/example_dags/example_kubernetes_decorator.py

```diff
@@ -0,0 +1,68 @@
+#
```

Comment:
Should this example live with the Kubernetes provider instead of core Airflow? 

Or since this example DAG is net-new, should this actually be AIP-47 compliant as a provider system test similar to the [`@task.docker` example](https://github.com/apache/airflow/blob/main/tests/system/providers/docker/example_taskflow_api_etl_docker_virtualenv.py)?

Reply:
> should this actually be AIP-47 compliant as a provider system test similar to the `@task.docker` example?

Oh yes. The example DAG was implemented way before AIP-47 was accepted (shows how long this dragged on…)

I’ll add a commit to convert it.